docs/installation.md

```diff
@@ -0,0 +1,45 @@
+# Installation
+
+To install this application you should search for "Elastic Agent" in the GCP Marketplace and install it through the GCP Console.
+
+From the product listing you can choose the version of the application to install. We maintain `7.17.x` and `8.x` stack versions.
+
+By clicking "Configure" a guided process starts to collect some deploymenta details.
+
+You will need to choose:
+- the GKE cluster where to deploy the application: you can select an existing cluster or create a new one;
+- the namespace within the GKE cluster; selecting `default` will deploy the Application to the `default` namespace and the Elastic Agent in the (`kube-system`);
+- the application instance name (this is the name of the deployed application in your GKE cluster);
+- the Service Account of the application; **do not change this field**, a dedicated Service Account with required permissions will be created;
+- Fleet Server URL to connect to; if left empty Kibana Host, Kibana Fleet Username and Kibana Fleet Password are needed;
+- Fleet enrollment token; is an Elasticsearch API key to enroll one or more Elastic Agents in Fleet. See: https://www.elastic.co/guide/en/fleet/current/fleet-enrollment-tokens.html. If left empty Kibana Host, Kibana Fleet Username and Kibana Fleet Password are needed;
+- Kibana Username; user name to connect to Kibana;
+- Kibana Fleet Password; password to connect to Kibana;
+- Kibana Host; URL to connect to Kibana;
+- Container resource request - CPU; the requested container CPU, depends on use case;
+- Container Resource Request - Memory; the requested container memory, depends on use case.
+
+Once the application is installed it should automatically enroll in Fleet. 
+
+At this point the Elastic Agent is ready to accept policies, but is not yet actively collecting any data, as there is no default policy.
+
+To collect data go to the Fleet UI and configure an Agent Policy.
```

Comment:
As I understand this may or may not be true? Say if the user already has an Agent policy and an associated Kubernetes integration and used its “token” wouldn’t the data start to get ingested automatically? I guess the real thing is what enrollment token has the user used? Feels like we need to provide some guidance to the users here? Can we expand on this in the document?

Reply:
Correct, I didn't know about this! This is also mentioned in our [docs](https://www.elastic.co/guide/en/fleet/current/fleet-enrollment-tokens.html):
> All Elastic Agents enrolled through this token will use the selected policy unless you assign or enroll them in a different policy.

I'll add guidance on setting up the Kubernetes integration and use that enrollment token.

Reply:
I prepared https://github.com/elastic/elastic-agent-gcp-kubernetes-app/pull/7 to clarify this aspect.